rero_mef/api.py

```
@@ -125,6 +125,8 @@ class EntityRecord(Record):
     object_type = "rec"
     name = None
 
+    _extensions = [MD5Extension(), SchemaExtension(), DeletedStateExtension()]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)